FABulous/fabric_generator/gds_generator/helper.py

```diff
@@ -100,10 +116,10 @@ def get_routing_obstructions(config: Config) -> list[tuple[int, int, int, int]]:
 
     parsed_obstructions = defaultdict(list)
     for obs in obstructions:
-        if len(obs) != 4:
+        if len(obs) != 5:
             raise ValueError(
                 f"Invalid obstruction {obs}. Each obstruction must be a tuple of "
-                "4 integers."
+                "metal layer flowed by 4 decimals"
```

Comment:
```suggestion
                "the metal layer name followed by 4 decimals"
```

I think you meant this (plus a short extension of the description)?